.github/workflows/build_test.yml

```
@@ -0,0 +1,69 @@
+name: Build and Test
+
+on: [pull_request]
+
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: evm # applied only for "run" steps
+    steps:
+      - name: Checkout
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+
+      - name: Set up Go
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        with:
+          cache-dependency-path: evm/go.sum
+          go-version-file: evm/go.mod
+
+      - name: Build
+        run: go build -v ./...
```

What does `-v ` add for us here?
```suggestion
        run: go build ./...
```